dev_docs/HOW_TO.md

```diff
@@ -58,18 +55,19 @@ See [apm-server TESTING.md](https://github.com/elastic/apm-server/blob/main/dev_
 
 ### 5. Finalize PRs
 
-1. Create a PR in apm-data, and have it reviewed and merged.
-2. In apm-server, bump apm-data dependency.
+1. If you have modified the `apm-data` plugin on ES, have that PR reviewed and merged first.
+2. Create a PR in apm-data, and have it reviewed and merged.
```

Comment:
```suggestion
2. Create a PR in apm-data, and have it reviewed and merged. Follow the instructions in the repo to release a new version, otherwise apm-server automation to bump apm-data version won't work.
```